admin/app/dfp/DfpDataCacheJob.scala

```diff
@@ -22,9 +22,11 @@ class DfpDataCacheJob(adUnitAgent: AdUnitAgent,
     log.info("Refreshing data cache")
     val start = System.currentTimeMillis
     val data = loadLineItems()
+    val sponsorshipLineItemIds = dfpApi.readSponsorshipLineItemIds()
     val duration = System.currentTimeMillis - start
     log.info(s"Loading DFP data took $duration ms")
     write(data)
+    Store.putNonRefreshableLineItemIds(sponsorshipLineItemIds)
```

Comment:
Is this the right place to put it? After the `write`?

Reply:
Putting it here because what's going on inside `write` is very confusing and depends on deriving pieces of data from other data, which is partially cached.  Just reading current sponsorship IDs is quite straightforward and shouldn't really be dependent on any other data.